docs/light/getting_started.md

```diff
@@ -1,40 +1,38 @@
-| trust-store | DIRECTORY | "$HOME/.lcd"            | false    | directory for save checkpoints and validator sets    |
 
-Sample command:
-
-```bash
-gaiacli light-client --chain-id=test --laddr=tcp://localhost:1317  --node tcp://localhost:46657 --trust-node=false
+| Parameter       | Type      | Default                 | Required | Description                                          |
+| --------------- | --------- | ----------------------- | -------- | ---------------------------------------------------- |
+| chain-id        | string    | null                    | true     | ID of chain we connect to, must be specified |
+| home            | string    | "$HOME/.gaiacli"        | false    | directory for config and data, such as key and checkpoint |
+| node-list       | string    | "tcp://localhost:26657" | false    | Full node list to connect to, example: "tcp://10.10.10.10:26657,tcp://20.20.20.20:26657" |
+| laddr           | string    | "tcp://localhost:1317"  | false    | Address for server to listen on |
+| trust-node      | bool      | false                   | false    | Trust full nodes or not |
+| swagger-host-ip | string    | "localhost"             | false    | The host IP of the Gaia-lite server, swagger-ui will send request to this host |
+| modules         | string    | "general,key,token"     | false    | Enabled modules |
+
+Sample command to start gaia-lite node:
+```
+gaiacli lite-server --chain-id {your chain id}
 ```
 
-## Gaia Light Use Cases
-
-LCD could be very helpful for related service providers. For a wallet service provider, LCD could
-make transaction faster and more reliable in the following cases.
-
-### Create an account
-
-![deposit](pics/create-account.png)
-
-First you need to get a new seed phrase :[get-seed](api.md#keysseed---get)
-
-After having new seed, you could generate a new account with it : [keys](api.md#keys---post)
-
-### Transfer a token
+When the connected full node is trusted, then the proof is not necessary, so you can run gaia-lite node with trust-node option:
+```
+gaiacli lite-server --chain-id {your chain id} --trust-node
+```
 
-![transfer](pics/transfer-tokens.png)
+If you have want to run gaia-lite node in a remote server, then you must specify the public ip to swagger-host-ip
+```
+gaiacli lite-server --chain-id {your chain id} --swagger-host-ip {remove server public ip}
+```
 
-The first step is to build an asset transfer transaction. Here we can post all necessary parameters
-to /create_transfer to get the unsigned transaction byte array. Refer to this link for detailed
-operation: [build transaction](api.md#create_transfer---post)
+The gaia-lite node can connect to multiple full nodes. Then gaia-lite will do load balancing for full nodes which is helpful to improve reliability and TPS. You can do this by this command:
+```
+gaiacli lite-server --chain-id {your chain id} --node-list tcp://10.10.10.10:26657,tcp://20.20.20.20:26657
```

Comment:
`gaiacli lite-server --chain-id=<chain_id> --node-list=tcp://10.10.10.10:26657,tcp://20.20.20.20:26657`. Although not sure if the node list needs to be in between " ". Will double check that

Reply:
I have double checked that. The node list doesn't need to be in between " "